activerecord/test/cases/tasks/database_tasks_test.rb

```diff
@@ -447,6 +447,29 @@ def test_creates_test_and_development_databases_when_rails_env_is_development
       ENV["RAILS_ENV"] = old_env
     end
 
+    def test_creates_development_database_without_test_databasne_when_skip_test_database
+      old_env = ENV["RAILS_ENV"]
+      ENV["RAILS_ENV"] = "development"
+      ENV["SKIP_TEST_DATABASE"] = "true"
```

Comment:
What were your thoughts on the approach suggested in https://github.com/rails/rails/issues/27299#issuecomment-618107586, so an extra environment variable doesn't need to be set, rather only the dev database will be used when  `RAILS_ENV=development` is explicitly set? This might be slightly simpler/more intuitive, though possibly this could be confusing as there is different behaviour when `RAILS_ENV` is explicitly vs implicitly set to `development`

Reply:
Please read the issue you linked completely. I explain why only setting `RAILS_ENV=development` is not enough.

Reply:
Where? I can't see a comment from you mentioning `RAILS_ENV` directly

Reply:
https://github.com/rails/rails/issues/27299#issuecomment-295536459
https://github.com/rails/rails/issues/27299#issuecomment-295536872

Reply:
Thanks, the former doesn't mention `RAILS_ENV` being explicitly set, but the latter reason makes sense. I've just been following the issue from a while back and so had forgotten about a linked commit message from 3 years ago when I got notified about the issue again.